### PR TITLE
Hotfix/1.32.3

### PR DESCRIPTION
**Pull request by 1saf**
Closed the previous one due to conflicting branch names.

# Description

Currently we have a problem with pool creation in that there is no recoverability mechanism built into the flow. This PR introduces that ability to recover previous state for the user to return to where they were before.

The main concept is that once the user reaches the PreviewPool stage of pool creation (the step where the buttons for creation and seeding are) the UI will save the pool creation state to the users local storage. The creation state has a new flag called `needsSeeding` which is set after a successful create action.

So the two main points of recovery are: 
- user reaches the final step of creation and doesn't create the pool. the state is saved and the user can return to this
- the user creates the pool but doesn't seed liquidity, or the seed liquidity transaction fails, the user can refresh or come back to this page and it'll recover and allow them to fund the pool

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other

## How should this be tested?

1. Go to pool creation, go through the flow as you would and once you reach the final step, exit out and open up the UI again. It should open up to the last step with the inputs you had selected before
2. Go through test 1, but this time create the pool - then exit out. Coming back to the UI it should go to the last step and the only action should be to fund the pool
3. 
## Checklist:

- [X] I have performed a self-review of my own code
- [X] I have commented my code where relevant, particularly in hard-to-understand areas
- [X] My changes generate no new console warnings
- [X] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

**Comment by 1saf** [comment]
Commits from "028c64e57576e7c858a742e837b09bf7b6126979" were somehow missing from develop

**Review by garethfuller — APPROVED** [review]
Looks good.

This isn't really to do with this PR but I've never actually seen the similar pools widget in the right context, only ever the flicker. See the screenshot as an example, this is me trying to create an 80/20 BAL/WETH pool on mainnet which I know there is a similar pool for. Where in the flow should I be seeing that there is a similar pool?
Screenshot 2021-12-09 at 12 03 29

File: src/components/cards/CreatePool/SimilarPools.vue [hunk 1]
```diff
@@ -30,6 +31,15 @@ const title = computed(() => {
...
 });
 
+// limit the similar pools on the UI to the first 4 ones
+// with highest tvl
+const relevantSimilarPools = computed(() => {
+  return take(
+    orderBy(similarPools.value, pool => Number(pool.totalLiquidity), 'desc'),
+    4
+  );
+});
```
[the remark this answers is not in the document]
Comment (1saf) [reply src/components/cards/CreatePool/SimilarPools.vue:41]: thats right. the custom subgraph query didnt work

File: src/components/cards/CreatePool/PoolSummary.vue [hunk 2]
```diff
@@ -142,15 +143,15 @@ const chartConfig = computed(() => {
  * WATCHERS
  */
 watch(
-  seedTokens,
+  [seedTokens, hasRestoredFromSavedState],
   async () => {
     const colors = await calculateColors();
     await nextTick();
     await nextTick();
     await nextTick();
```
[the remark this answers is not in the document]
Comment (1saf) [reply src/components/cards/CreatePool/PoolSummary.vue:151]: mightve been a dirty merge. will cleanup

**Pull request merged**